.editorconfig

```diff
@@ -0,0 +1,26 @@
+# This file is for unifying the coding style for different editors and IDEs
+# editorconfig.org
+# https://github.com/editorconfig/editorconfig/wiki/EditorConfig-Properties
+
+# WordPress Coding Standards
+# http://make.wordpress.org/core/handbook/coding-standards/
+
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
```

Comment:
From the editorconfig docs:

>end_of_line
> Line ending file format (Unix, DOS, Mac)
>The values are case insensitive. They will be lowercased by the core library.

>NOTE: if you want to use native line endings between different operating systems it is better not to set this option and leave that task to the VCS! In the future we might add a value like native for this scenario (cf #226).

So maybe should be omitted?

Reply:
For PHP it shouldn't matter but since v2.7 is will be required for JS because of the DFV structure templates templates. (@pglewis)